onadata/apps/api/tests/views/test_user_permissions.py

```diff
@@ -83,6 +83,9 @@ def test_manager_can_update_xform(self):
 
         self.assertFalse(self.xform.shared)
 
+        data.pop('enketo_preview_url')
```

Comment:
This change is related to this issue [here](https://code.djangoproject.com/ticket/30024)

Reply:
This makes sense. Do we add the link - https://code.djangoproject.com/ticket/30024 - as a comment just above the `<obj>.pop('<key>')` so that in future, we can easily know why we had to do this?

Reply:
Yes, I'll add a link later on in case someone works on the code later on.